pkgs/by-name/ea/easysnap/package.nix

```
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation {
   pname = "easysnap";
-  version = "0-unstable-2022-06-03";
+  version = "0-unstable-2025-05-20";
 
   src = fetchFromGitHub {
     owner = "sjau";
     repo = "easysnap";
-    rev = "5f961442315a6f7eb8ca5b705bd52fe1e6d7dc42";
-    sha256 = "sha256-jiKdpwuw0Oil0sxUr/3KJ6Nbfxh8DvBei0yy0nRM+Vs=";
+    rev = "f9b2b65e5469f295bdd9515d464ce709c969c95a";
+    sha256 = "sha256-+1zXypl3fcF2Y/VnLBz+T8Mf94Eu7bqnqoHEjIUpy+U=";
```

```suggestion
    hash = "sha256-+1zXypl3fcF2Y/VnLBz+T8Mf94Eu7bqnqoHEjIUpy+U=";
```
use hash when sri hash